content/docs/getting-started/license.mdx

```diff
@@ -5,7 +5,7 @@ description: Learn more about the open-source licensing rights of the Flowbite R
 
 ## MIT License
 
-Copyright &copy; 2023 Bergside Inc.
+Copyright &copy; 2024 Bergside Inc.
```

Comment:
Stupid question... Can't it be dynamic?

Reply:
Not stupid at all, I already thought of this and it seemed weird to put a JS snippet there for the readers that open it in github or any `mdx` viewer.

Reply:
Just checked some other big repos and it seems like not even `tailwindcss` includes the hardcoded year 🤔

Reply:
Well, it should be possible to make it dynamic somehow. I can't imagine creating a PR every single year just to update the year at the footer. 🤣

Reply:
so footer problem is gone, we're talking here only `license.mdx` file 😄 which is in docs

Reply:
Yeah, got confused by the image. LOL